app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers");
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
+
+
+app.use((err, req, res, next) => {
```

Comment:
This is picky but we dont need to use this error handler yet as there's nothing to check for, be carful to only write the code you need to pass the tests that you are doing 👍